src/AbstractController.php

```diff
@@ -125,7 +126,7 @@ public function dispatch($action = null, array $actionArgs = [])
         }
 
         // Call action method
-        $actionResponse = call_user_func_array([$this, $actionMethod], $collectedArgs);
+        $actionResponse = \call_user_func_array([$this, $actionMethod], $collectedArgs);
```

Comment:
Any reasoning behind which function calls to make fully qualified? Seems a bit inconsistent.

Reply:
Do some real app benchmarking before you refactor everything though, because it does make the code less readable. Would be interesting to see if any of the app methods are called enough times during a request cycle to make any real world difference.

Reply:
There difference isn't huge, and is of course tiny compared to things like database calls.

There's another option, however, which is to add a `declare(no_dynamic_functions=1)` to the start of a file.

Reply:
`no_dynamic_functions` is only an unofficial suggestion as far as I know. It does not exist yet.

Reply:
Ah, my mistake - I misread an article.